NICE.Identity.Management/ClientApp/src/views/UsersList/UsersList.tsx

```diff
@@ -1,42 +1,103 @@
-import React, { FunctionComponent } from "react";
+import React, { Component } from "react";
+
 import { Link } from "react-router-dom";
-import { useFetch } from "../../hooks/hooks";
+import { Breadcrumbs, Breadcrumb } from "@nice-digital/nds-breadcrumbs";
+import { Card } from "@nice-digital/nds-card";
+import { Tag } from "@nice-digital/nds-tag";
+
+import { Endpoints } from "../../data/endpoints";
+
 import { Filter } from "../../components/Filter/Filter";
 
-export const UsersList: FunctionComponent = () => {
-	const [data, isLoading] = useFetch(
-		"https://www.json-generator.com/api/json/get/cfQXOXbwgO?indent=2",
-	);
-
-	return (
-		<div className="grid">
-			<div data-g="3">
-				<Filter />
-			</div>
-			<div data-g="9">
-				{isLoading ? (
-					<p>Loading...</p>
-				) : (
-					<table>
-						<thead>
-							<tr>
-								<th>Name</th>
-								<th>Email</th>
-							</tr>
-						</thead>
-						<tbody>
-							{data.map(({ _id, name, email }) => (
-								<tr key={_id}>
-									<td>
-										<Link to={`/users/${_id}`}>{name}</Link>
-									</td>
-									<td>{email}</td>
-								</tr>
-							))}
-						</tbody>
-					</table>
-				)}
-			</div>
-		</div>
-	);
+type User = {
+	id: number;
+	email_address: string;
+	user_id: string;
+	first_name: string;
+	last_name: string;
+	email_verified: boolean;
+	blocked: boolean;
 };
+
+interface StateType {
+	data: Array<User>;
+}
+
+interface PropsType {}
+
+export class UsersList extends Component<PropsType, StateType> {
+	constructor(props: PropsType) {
+		super(props);
+		this.state = {
+			data: [],
+		};
+	}
+
+	fetchData = async (url: string) => {
+		const response = await fetch(url);
+		const data = await response.json();
+		this.setState({ data });
+	};
+
+	componentDidMount() {
+		this.fetchData(Endpoints.usersList);
+	}
+
+	render() {
+		const { data } = this.state;
+
+		return (
+			<>
+				<Breadcrumbs>
+					<Breadcrumb to="/" tag={Link}>
+						Home
+					</Breadcrumb>
+					<Breadcrumb>Users</Breadcrumb>
+				</Breadcrumbs>
+
+				<div className="page-header">
+					<h1 className="page-header__heading">Users</h1>
+				</div>
+
+				<div className="grid">
+					<div data-g="12 md:3">
+						<Filter />
+					</div>
+					<div data-g="12 md:9">
+						{!data.length ? (
```

Comment:
I think we're probably missing some `aria-live` and `aria-busy` attributes because otherwise a screenreader wouldn't get notified that the data has loaded